src/app/src/api/submissions.js

```diff
@@ -0,0 +1,48 @@
+import api from './api';
+import TAGS, {
+    getListTagProvider,
+    getNewItemTagInvalidator,
+    getSingleItemProvider,
+    getUpdateItemTagInvalidator,
+} from './tags';
+
+const submissionApi = api.injectEndpoints({
+    endpoints: build => ({
+        getDraftSubmissions: build.query({
+            query: () => ({
+                url: '/submissions',
+                method: 'GET',
+                params: { type: 'drafts' },
```

Comment:
Probably not the best URL scheme: `/submissions?type=drafts`, but I included it to demonstrate how query parameters can be used.